dataengineeringutils/glue.py

```diff
@@ -77,7 +77,7 @@ def overwrite_or_create_database(db_name, db_description=""):
         pass
 
     log.debug("Creating database: {}".format(db_name))
-    response = glue_client.create_database(**db)
```

Comment:
I'm now returning the response here deliberately in case we want it for anything